src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -153,12 +169,16 @@ ucs_status_t uct_rdmacm_cm_ep_conn_param_init(uct_rdmacm_cm_ep_t *cep,
     if (priv_data_ret < 0) {
         ucs_assert(priv_data_ret > UCS_ERR_LAST);
         status = (ucs_status_t)priv_data_ret;
-        ucs_error("rdma_cm private data pack function on ep %p failed with error: %s",
+        ucs_error("%s: rdma_cm private data pack function on ep %p failed with error: %s",
```

Comment:
the flags should not be printed as 1st arg, the main object here is "cep"
add new function to convert the whole cep to string like this:
```
 ucs_error("%s: rdma_cm private data pack function failed with error: %s",
                 uct_rdmacm_cm_ep_str(cep, ep_str, UCT_RDMACM_EP_STR_LEN), ucs_status_strin(status));
and
const char* uct_rdmacm_ep_str(uct_rdmacm_cm_ep_t *cep, char *str,
                                    size_t max_len)
{
   char buf[UCT_RDMACM_EP_STR_LEN];
   ucs_flags_str(buf, sizeof(buf), cep->ep_flags, ep_flag_to_str);
   ucs_snprintf_safe(str, max_len, "rdmacm_ep %p flags %s:" buf);
   return str;
}
```